fhi_users/auth/rest_auth_views.py

```
@@ -773,14 +792,21 @@
                 return Response(
-                    serializers.StatusResponseSerializer(
-                        {"status": "failure", "message": "Reset token has expired"}
+                    common_serializers.ErrorSerializer(
+                        {"error": "Reset token has expired"}
                     ).data,
                     status=status.HTTP_400_BAD_REQUEST,
                 )
 
             # Update password
             user = reset_token.user
+            if not auth_utils.validate_password(data["new_password"]):
```

Ruff (0.8.2)

Undefined name `auth_utils`

(F821)